Text/src/test/java/org/Marta/TextUtilsTest.java

```diff
@@ -0,0 +1,49 @@
+package org.Marta;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class TextUtilsTest {
```

Comment:
A gdzie ten program, który z tych wszystkich metod zrobi to co tutaj w treści zadania jest? :D 

```
Napisz program, który operuje na zadanym tekście i potrafi :

sprawdzić, czy tekst zawiera jakiekolwiek znaki
jeśli punkt 1 jest prawdziwy, to sprawdzić czy łańcuch znaków jest palindromem
jeśli punkt 1 jest prawdziwy, to obliczyć długość tekstu
```

Reply:
w sensie mam dodać maina i dać użytkownikowi wprowadzić tekst, a potem wyświetlić wyniki tych metod? ;)